examples/etcd/etcd-in-cluster/README.md

```diff
@@ -0,0 +1,323 @@
+
+NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/nginx   1/1     1            1           40s
+
+NAME                               DESIRED   CURRENT   READY   AGE
+replicaset.apps/nginx-86c57db685   1         1         1       40s
+```
+
+## Protect the Application
+
+We can now take snapshot of the ETCD server that is running by creating backup actionset that is going to execute backup phase from the blueprint that we have
+created above
+
+**Note**
+Pleae make sure to change the **profile name**, the **ETCD pod name**, **pod namespace** and **blueprint name** in the `backup-actionset.yaml` manifest file.
+
+```
+# find the profile name
+» kubectl get profile -n kanister
+NAME               AGE
+s3-profile-nnvmm   54s
+
+# fine blueprint name
+» kubectl get blueprint -n kanister
+NAME             AGE
+etcd-blueprint   41s
+
+# create actionset
+» kubectl create -f backup-actionset.yaml -n kanister
+actionset.cr.kanister.io/backup-hnp95 created
+
+# you can check the status of the actionset by describing it and making sure that it is succeeded
+» kubectl describe actionsets.cr.kanister.io -n kanister backup-hnp95
+
+# You should see events like this
+...
+...
+Events:
+  Type    Reason           Age   From                 Message
+  ----    ------           ----  ----                 -------
+  Normal  Started Action   34s   Kanister Controller  Executing action backup
+  Normal  Started Phase    34s   Kanister Controller  Executing phase takeSnapshot
+  Normal  Ended Phase      34s   Kanister Controller  Completed phase takeSnapshot
+  Normal  Started Phase    34s   Kanister Controller  Executing phase uploadSnapshot
+  Normal  Ended Phase      0s    Kanister Controller  Completed phase uploadSnapshot
+  Normal  Started Phase    0s    Kanister Controller  Executing phase removeSnapshot
+  Normal  Ended Phase      0s    Kanister Controller  Completed phase removeSnapshot
+  Normal  Update Complete  0s    Kanister Controller  Updated ActionSet 'backup-hnp95' Status->complete
+```
+
+Once the backup actionset is complete, we can check the object storage to make the backup is uploaded successfully.
+
+## Imitate Disaster
+
+Now let's assume something went wrong and we lost the test namespace that we created eariler. Please delete the namespace manually
+
+```
+» kubectl delete ns nginx
+namespace "nginx" deleted
+
+» kubectl get all -n nginx
+No resources found in nginx namespace.
+```
+
+## Restore the ETCD cluster
+
+Create restore action using below command
+```
+kanctl --namespace kanister create actionset --action restore --from backup-hnp95
+actionset restore-backup-hnp95-dcwh9 created
+```
+
+And you will be able to see below message in the kanister operator's log pod
+
+```
+Automated restore for ETCD is not supported please follow the examples docs to restore the backup manually. Backup path : etcd_backups/kube-system/etcd-ubuntu-s-4vcpu-8gb-blr1-01-master-1/2020-08-07T11:21:23Z/etcd-backup.db.gz
```

Comment:
Instead of checking Kanister operator pod logs, can we set the backup path as an output artifact in restore action and ask user (here in docs) to find the backup path by describing the actionset?

Reply:
@viveksinghggits I think we should remove this block.
The drawbacks with this approach are:
- User has to check logs of pods each time they do a restore
- If there are multiple backups, its becomes hard to figure out the correct backup path for the restorepoint
So, I think the correct way is to set backpath in output artifact of restore action. So that if someone want to restore to specific restorepoint, they can just describe the restore actionset to see the backuppath in the artifacts